### DISCUSSION TITLE
How can I identify the list of native APIs in a DLL file?

### DISCUSSION THREAD

**Discussion by Dinesh0723** [eclipse-platform/eclipse.platform.swt#1175] [Q&A]
How can I able to find the list of native APIs in a DLL (For an example, consider swt-win32-4965r3.dll) file?

**Comment by fedejeanne** [comment]
You mean something like this?

![image](https://github.com/eclipse-platform/eclipse.platform.swt/assets/2205684/f9044082-0b07-4a8c-896c-caeeea2e039b)

That's the [DLL export viewer](http://www.nirsoft.net/utils/dll_export_viewer.html)
